### ISSUE TITLE
Edge support

### ISSUE THREAD

**Issue by broerse** [ember-cli/ember-twiddle#121]
ember-twiddle should be made compatible with the new Windows 10 Edge browser.

**Comment by broerse** [comment]
It seems we need something like this to do it: https://github.com/jugglinmike/srcdoc-polyfill/blob/master/srcdoc-polyfill.js

**Comment by broerse** [comment]
Yes! There is now an official npm version. Perhaps we can include it now?

https://www.npmjs.com/package/srcdoc-polyfill

**Comment by Gaurav0** [comment]
@broerse Does it work with sandboxed iframes?

**Comment by broerse** [comment]
@Gaurav0 I am not an expert but this `sandbox="allow-scripts allow-same-origin"` http://jsfiddle.net/wUvrF/1/ works in Edge.

**Comment by Gaurav0** [comment]
We cannot allow-same-origin for security reasons.

**Comment by broerse** [comment]
As I said I am not the expert. Perhaps it is not needed. Can we create a test I can run on Edge?

**Comment by joostdevries** [comment]
@broerse A pr with the srcdoc polyfill for edge would be welcome. The sandboxing policy cannot be changed for security reasons. I cannot test in Edge myself and we haven't got anything like Sauce Labs setup for that (yet) so we would have to rely on acceptance testing by yourself.

**Comment by broerse** [comment]
@joostdevries I can always do the acceptance testing. I am not sure I can write the PR but will try to find some time. If someone likes to help with this contact me.

**Comment by jdalton** [comment]
If you all create a test I can check it out.

**Comment by Gaurav0** [comment]
Update: Edge now supports `srcdoc` but Ember Twiddle still blocks it. See https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12375527/
